gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/medium_tests/gnomad_genomes_GRCh37_chrX_head2500_run_vep.json

```diff
@@ -12,26 +8,25 @@
     "runner": "DataflowRunner",
     "run_annotation_pipeline": "True",
     "annotation_output_dir": "gs://integration_test_runs/temp/vep_output/{TABLE_NAME}",
-    "disk_size_gb": 150,
-    "num_workers": 1,
+    "num_workers": 2,
     "assertion_configs": [
       {
         "query": ["NUM_ROWS_QUERY"],
-        "expected_result": {"num_rows": 9953}
+        "expected_result": {"num_rows": 2337}
       },
       {
         "query": [
           "SELECT COUNT(0) AS num_annotation_sets ",
           "FROM {TABLE_NAME} AS T, T.alternate_bases AS A, A.CSQ AS CSQ"
         ],
-        "expected_result": {"num_annotation_sets": 45770}
+        "expected_result": {"num_annotation_sets": 21167}
       },
       {
         "query": [
           "SELECT COUNT(0) AS num_annotation_sets ",
           "FROM {TABLE_NAME} AS T, T.alternate_bases AS A, A.CSQ_VT AS CSQ_VT"
         ],
-        "expected_result": {"num_annotation_sets": 45770}
+        "expected_result": {"num_annotation_sets": 21059}
```

Comment:
Did you managed to investigate why these numbers are different between CSQ and CSQ_VT? We had some theories but I wonder if you prove/disprove them?

Reply:
I didn't look into it on Friday, but I can check it out when I get back!